feat: updating limitations of Reactors (#321)
Basis-Theory/developers.basistheory.com: docs/api/reactors/reactors.mdx

| `tokens`      | _object_ | (Optional) [Token(s)](/docs/api/tokens#token-object) created from the `tokenize` block of the Reactor code [response](#reactor-code-response-object) |
| `raw`         | _object_ | (Optional) Raw output returned from the Reactor                                                                                                                                               |

### Limitations

| Limit        | Note                                                                                                                                                                                   |
| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>BT</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |

Reviewer: > You could pull in more data using the injected <code>BT</code> client from within the Reactor code.

What do you mean? Can you give an example?

Author: I just copied this from our message there for a few months, haha. I don't know how to explain more. I think removing that might be the best or trying to think of a way to make it more broad - I don't think many people have this problem and if they do I think we've really only seen people pulling in non-token data.

Thoughts

```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. <a href="https://basistheory.com/contact">Please reach out</a> if you need help using larger datasets. |
```

Reviewer: Yea, let me make another suggestion to suit that better.

> | Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>BT</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |

Reviewer: ```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>bt</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |
```

> | Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>bt</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |

Reviewer: ```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. <a href="https://basistheory.com/contact">Please reach out</a> if your use case requires more than that. |
```
